tests/tests/swfs/avm2/bitmapdata_histogram/Test.as

```diff
@@ -0,0 +1,42 @@
+package {
+    import flash.display.Sprite;
+    public class Test extends Sprite {
+        public function Test() { }
+    }
+}
+
+import flash.display.BitmapData;
+import flash.geom.Rectangle;
+
+function print(histogram: *) {
+    trace("");
+    trace("r " + histogram[0]);
+    trace("g " + histogram[1]);
+    trace("b " + histogram[2]);
+    trace("a " + histogram[3]);
+}
+
+var bitmap = new BitmapData(10, 10);
+print(bitmap.histogram());
+
+bitmap.setPixel32(0, 0, 0xAABBCCDD);
```

Comment:
just to be sure, can you also test a pixel value that shows premultiplied precision loss? (like: A=0x20, R=6, and precision loss on roundtrip turning R to 8)

Reply:
I think there is already a precision loss here. `getPixel32(0,0)` returns `0xAABBCCDC`.